hardware/sim/run_sim

```diff
@@ -364,7 +364,19 @@
      prc[$i]=0
      if [ "$PAR" == "1" ];then STIMLOG="stim.log";else STIMLOG="stim${i}.log";fi
      if [ "$TST_NAME" == "XTERM" ];then
-       xterm -title "testcase window, use >script ${STIMLOG}< to log input" & 	# tee+RC doesnt work here, use script instead
+        if [ -z "$TMUX" ];then
+          xterm -title "testcase window, use >script ${STIMLOG}< to log input" & 	# tee+RC doesnt work here, use script instead
+        else
+          # Running inside a tmux session
+          tmux new-window -c $SIMBASE/$SIMDIR/$SIMOUT
```

Comment:
tmux: unknown option -- c
usage: new-window [-adk] [-n window-name] [-t target-window] [command]

Reply:
You seem to be using a version of tmux that is older than 1.9 (can be checked with `tmux -V`).
The [version 1.9 release notes](https://github.com/tmux/tmux/blob/1.9/CHANGES) indicate the introduction of the `-c` flag:
```
CHANGES FROM 1.8 TO 1.9, 20 February 2014

Incompatible Changes
====================
[...]
* 'default-path' has been removed.  The new-window command accepts '-c' to
  cater for this.  The previous value of "." can be replaced with: 'neww -c
  $PWD', the previous value of '' which meant current path of the pane can
  be specified as:  'neww -c "#{pane_current_path}"'
```

I'm using Ubuntu 16.04, which provides tmux 2.1. Which operating system are you using?

Reply:
Damn you are right...tmux 1.6. Let me see if I can move to a newer ubuntu release. Thx

Reply:
I have only the ability to have a tmux 1.8 so i can only rely on your code.;-)